data/coco128.yaml

```
@@ -10,8 +10,10 @@
 download: https://github.com/ultralytics/yolov5/releases/download/v1.0/coco128.zip
 
 # train and val data as 1) directory: path/images/, 2) file: path/images.txt, or 3) list: [path1/images/, path2/images/]
-train: ../coco128/images/train2017/  # 128 images
-val: ../coco128/images/train2017/  # 128 images
+
+bucket: #Insert bucket name
```

This is a very small stylistic choice, but I feel that something like

```suggestion
bucket: your-bucket-here
```

is a better way of describing the sentiment. This way the config actually has a value, even if it's clearly a dummy-value.

---

Fixed a33490a6a6b0d8860a9e67280bf0972a23f022c3